plugins/inventory/xen_orchestra.py

```diff
@@ -138,21 +139,41 @@ def create_connection(self, xoa_api_host):
         self.conn = create_connection(
             '{0}://{1}/api/'.format(proto, xoa_api_host), sslopt=sslopt)
 
+    CALL_TIMEOUT = 100
+    """Number of 1/10ths of a second to wait before method call times out."""
+    def call(self, method: str, params: dict) -> dict:
+        """Calls a method on the XO server with the provided parameters."""
+        id = int(self.pointer)
```

Comment:
It is already an int.
```suggestion
        id = self.pointer
```

Reply:
done :)